src/index.ts

```diff
@@ -343,9 +345,22 @@ export function mande(
     patch: _fetch.bind(null, 'PATCH'),
 
     // these two have no body
-    get: (url: string, options?: Options) => _fetch('GET', url, null, options),
-    delete: (url: string, options?: Options) =>
-      _fetch('DELETE', url, null, options),
+    get: (
```

Comment:
rather than updating these, I think this should doable only changing the types of `_fetch`

Reply:
okay, I'll take a look.